### DISCUSSION TITLE
Is there a documented way to convert from RGB/HEX to OKLCH?

### DISCUSSION THREAD

**Discussion by eduard-buta** [evilmartians/oklch-picker#142] [Q&A]
Hi there. I'm trying to write a script for Adobe Illustrator, and I want to use the OKLCH color space for something. I can't find any resources online on how to convert standard color spaces to OKLCH. Would appreciate any guidance!

**Comment by ai** [comment]
https://gist.github.com/dkaraush/65d19d61396f5f3cd8ba7d1b4b3c9432
